### PR TITLE
fix #66 update readme

### LINKED ISSUE #66: Update Readme
Currently `Readme`  doesn't contain much information about the project. 
It should be clear what the project does without actually running the project

### PR DESCRIPTION
**Pull request by prybalko**
Add useful information to README about the project.

The description was taken from 
https://docs.google.com/document/d/1eQ6VaKVs3Sn9we2fs2NRAljALsJd6BH12rGGhwAUAX0/edit#

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,76 @@
...
-![Yarn tests](https://github.com/paritytech/banana_split/workflows/Yarn%20tests/badge.svg)
+Banana Split makes your paper backups more resilient and secure using [Shamir’s secret sharing](https://en.wikipedia.org/wiki/Shamir%27s_Secret_Sharing).
 
-[![Join the chat at https://gitter.im/paritytech/banana_split](https://badges.gitter.im/paritytech/banana_split.svg)](https://gitter.im/paritytech/banana_split?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+It splits your secret (e.g., paper backup) into many pieces, but only requires a majority of those pieces (N/2+1) to recover your secret.
 
-## Project setup
+## Why Banana Split?
+There’s a few problems with normal paper backups:
+If you have the whole paper backup somewhere, you’re vulnerable to having someone finding it and stealing it—without you ever knowing they did (e.g., they can just take a photo of it).
```
Comment (TarikGul) [inline README.md:9]:
```suggestion
If you have the whole paper backup somewhere, you’re vulnerable to someone finding it and stealing it—without you ever knowing they did (e.g., they can just take a photo of it).
```

**Review by TarikGul — DISMISSED** [review]
LGTM, some small suggestions. 

It might also be worth noting the cryptography used for encryption. For example: `scrypt` is used for key derivation, and `xsalsa20-poly1305` is used for encryption-authentication.

**Review by kirushik — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1,30 +1,76 @@
-# Banana Split
+# Banana Split 🍌 &middot; ![Yarn tests](https://github.com/paritytech/banana_split/workflows/Yarn%20tests/badge.svg) [![Join the chat at https://gitter.im/paritytech/banana_split](https://badges.gitter.im/paritytech/banana_split.svg)](https://gitter.im/paritytech/banana_split?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-![Yarn tests](https://github.com/paritytech/banana_split/workflows/Yarn%20tests/badge.svg)
+Banana Split makes your paper backups more resilient and secure using [Shamir’s secret sharing](https://en.wikipedia.org/wiki/Shamir%27s_Secret_Sharing).
 
-[![Join the chat at https://gitter.im/paritytech/banana_split](https://badges.gitter.im/paritytech/banana_split.svg)](https://gitter.im/paritytech/banana_split?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+It splits your secret (e.g., paper backup) into many pieces, but only requires a majority of those pieces (N/2+1) to recover your secret.
```
[the remark this answers is not in the document]
Comment (prybalko) [reply README.md]: I'll make it `into N pieces`. I think it implies that `N` is configurable

File: README.md (hunk 1 above)
Comment (prybalko) [reply README.md]: Thanks, fixed

**Comment by prybalko** [comment]
> It might also be worth noting the cryptography used for encryption. For example: `scrypt` is used for key derivation, and `xsalsa20-poly1305` is used for encryption-authentication.

I'm not sure whether it's an important product feature or a minor implementation detail. 
cc: @kirushik

**Review by TarikGul — APPROVED** [review]

**Review by vas3k — APPROVED** [review]

**Pull request merged**